Library/Homebrew/cleanup.rb

```diff
@@ -48,7 +48,7 @@ def prune?(days)
     end
 
     def stale?(scrub = false)
-      return false unless file?
+      return false unless file? || (symlink? && resolved_path.file?)
```

Comment:
Can this be an `if` instead? Better still, make it a separate `return` if possible.

Reply:
Can't be separate because it is an OR.  And `return if !file? && !(symlink? && resolved_path.file?)` isn't more readable.

Reply:
Sorry, missed this. I do genuinely find that more readable. An alternative would be to split the `symlink? && resolved_path.file?` into a separate variable.